.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+.vscode
```

Comment:
your IDE settings should not be checked into the repo, they are for your local project only.

Upon creating a new project, add a .gitignore file in the root of your project. Here, you add any directories/files that are intended for your local project only. That is ( for example ): .vscode ( your IDE settings ), .idea ( my IDE settings ). 

Other things you would typically add here is your .node_modules directory, that is created after you install packages from npm for example through a package.json file.